appveyor.yml

```diff
@@ -14,21 +16,25 @@ matrix:
 # what combinations to test
 environment:
   matrix:
-    - nodejs_version: 8
+    - nodejs_version: 10
 
 install:
   - ps: Install-Product node $env:nodejs_version x64
-  - npm install -g npm@4
   - set PATH=%APPDATA%\npm;%PATH%
+  - npm config set python 'C:\Python27\python.exe'
   - npm install
 
 build: off
 
 test_script:
   - node --version
   - npm --version
-  - cmd: npm test
+  - npm test
 
 deploy_script:
-  - IF "%APPVEYOR_REPO_TAG%" == "true" (npm run release)
-  - IF NOT "%APPVEYOR_REPO_TAG%" == "true" (echo 'Not tagged, skipping deploy')
+  - node --version
+  - npm --version
+  - npm run build:installer
```

Comment:
`npm run build:installer` generates the native installer for the current platform (in the case of an Appveyor VM, the Windows `'.exe'` installer is created).